lib/internal/test_runner/reporter/tap.js

```diff
@@ -32,6 +33,8 @@ async function * tapReporter(source) {
   for await (const { type, data } of source) {
     switch (type) {
       case 'test:fail': {
+        // If test bailed out do not report
```

Comment:
so a test can run without us reporting it? I am not sure this is good

Reply:
The test has error `kBailedOut` because has been canceled, I dont want to see canceled tests in the reporter